### ISSUE TITLE
replace "system-docker inspect docker env" w/ "system-docker inspect docker"

### ISSUE THREAD

**Issue by sols1** [rancher/os#1258]
**RancherOS Version: (ros os version)** 0.6.1

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)** does not matter

https://github.com/rancher/os/blob/master/tests/http_proxy_test.go#L12

```
-sudo system-docker exec docker env | grep NO_PROXY=invalid
+sudo system-docker inspect docker env | grep HTTP_PROXY=invalid
+sudo system-docker inspect docker env | grep HTTPS_PROXY=invalid
+sudo system-docker inspect docker env | grep NO_PROXY=invalid
```

**Comment by joshwget** [comment]
Yeah, I definitely agree with this change. If you'd like you can put in a PR, or I'll update it soon.

**Comment by sols1** [comment]
Fixed by #1329
